policyengine_api/services/policy_service.py

```diff
@@ -18,3 +59,134 @@ def get_policy_json(self, country_id, policy_id):
         except Exception as e:
             print(f"Error getting policy json: {str(e)}")
             raise e
+
+    def set_policy(
+        self, country_id: str, label: str, policy_json: dict
+    ) -> tuple[int, str, bool]:
+        """
+        Insert a new policy into the database
+
+        Arguments
+            country_id: str -- the country ID
+            label: str -- the policy label
+            policy_json: dict -- the policy content
+
+        Returns
+            tuple[int, str, bool] -- the new policy ID, a message, and whether or not
+            the policy already existed
+        """
+        print("Setting new policy")
+
+        try:
+
+            policy_hash = hash_object(policy_json)
+            api_version = COUNTRY_PACKAGE_VERSIONS.get(country_id)
+            # Check if policy already exists
+            print("Checking if policy exists")
+            existing_policy = self._get_unique_policy_with_label(
+                country_id, policy_hash, label
+            )
```

Comment:
**question** What purpose does api_version play here? Asking because we store it in the database, but not in the hash.